.github/workflows/pr-gate.yml

```
@@ -1,20 +1,26 @@
 name: PR Gate
+
 on:
   pull_request:
-    branches: [ master, main, develop, release-*, develop-go ]
+    types: [opened, reopened, synchronize]
+    # Gate only PRs targeting develop-go for now. Add branches here as each
+    # is ready to be gated (e.g. develop, release-*), or drop the filter to
+    # gate every PR.
+    branches: [ develop-go ]
```

YAMLlint (1.37.1)

[error] too many spaces inside brackets

(brackets)